tests/test.cpp

```diff
@@ -4572,6 +4572,61 @@ void JsonUnsortedArrayTest()
   TEST_NOTNULL(monster->testarrayoftables()->LookupByKey("ccc"));
 }
 
+void VectorSpanTest() {
+  flatbuffers::FlatBufferBuilder builder;
+
+  auto mloc =
+      CreateMonster(builder, nullptr, 0, 0, builder.CreateString("Monster"),
+      builder.CreateVector<uint8_t>({ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9 }));
+
+  FinishMonsterBuffer(builder, mloc);
+
+  auto monster = GetMonster(builder.GetBufferPointer());
+  auto mutable_monster = GetMutableMonster(builder.GetBufferPointer());
+
+  TEST_NOTNULL(monster->inventory());
+
+  {  // using references
+
+    auto const_inventory = flatbuffers::make_span(*monster->inventory());
```

Comment:
could you use the actual type instead of auto? It makes reading the test easier.

Same for the following uses.

Reply:
And this showed that I was using the wrong accessors - inventory() instead of mutable_inventory().